app/auth/forms.py

```diff
@@ -18,13 +18,14 @@ class PasswordStrength(object):
     '''
     def __init__(self, message=None):
         if not message:
-            message = 'Your password is too weak. Please try again with a more secure password.'
+            message = 'Your password is too easy to guess. Please try again\
```

Comment:
I can't see how this will work - am pretty sure you need quotes at the start and end of each line. See: https://stackoverflow.com/a/10660443